play-ebean/src/main/java/play/db/ebean/EbeanDynamicEvolutions.java

```diff
@@ -65,15 +66,11 @@ public void create() {
             }
             File evolutions = environment.getFile("conf/evolutions/" + key + "/1.sql");
             try {
-                String content = "";
-                if (evolutions.exists()) {
-                    content = new String(Files.readAllBytes(evolutions.toPath()), "utf-8");
-                }
-
+                String content = evolutions.exists() ? Files.readString(evolutions.toPath()) : "";
```

Comment:
IDEA still suggests replace with non-existent method after changing JDK to 1.8. Nice bug